rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,602 @@
+                view.addModel(pdb,'pdb')
+                # Protein Drawing style and colors
+                view = handleProteinStyle(uid, view, pModelNum)
+                
+        # zoomTo does not work well for surface and label... so, zoomTo should not be default settings
+        #view.zoomTo()
+        view.setBackgroundColor(globals()['background_'+uid].value)
+        
+        display(view.update())
+        
+    
+def handle_change(change):
+    """This function handles all the interactive widgets except buttons and 3Dmol.js viewer"""
+    update3D(change.owner._model_id)
+    
+def handle_start_button(b):
+    """This function handles start button."""
+    b.icon='check'
+    b.description="Done!"
+    uid=globals()['ipy_wgs'][b._model_id]
+    globals()['rdkit_3dviewer_start_button_clicked_'+uid] = True
+    update3D(b._model_id)
+    
+def handle_zoomTo_button(b):
+    """This function handles zoomTo button"""
+    uid=globals()['ipy_wgs'][b._model_id]
+    globals()['view_'+uid].zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+def ChangeActiveLigand(uid, molId, confId, keepExistingModels = True):
+    """This function handles ligand select through python code
+    uid : string 
+    molId : string 
+    confId : intiger
+    keepExistingModels : logical (if False - all models are removed - molId and confId associated model rendered)
+    """
+    if type(uid) != 'str':
+        uid = str(uid)
+    if type(molId) != 'str':
+        molId = str(molId)
+        
+    if globals()['molId_'+uid].value != molId:
+        globals()['molId_'+uid].value = molId
+        
+    if globals()['confId_'+uid].value != confId:
+        globals()['confId_'+uid].value = confId
+    
+    if keepExistingModels is False:
+        globals()['selectMultiMols_'+uid].value=False
+        globals()['selectAllMols_'+uid].value=False
+        globals()['selectMultiConfs_'+uid].value=False
+        globals()['selectAllConfs_'+uid].value=False
+    
+    update3D(globals()['molId_'+uid]._model_id)
+    
+    
+    
+def ShowConformers3D(uid = None,
+                     moldict = None, protein = None,
+                     useDrawAs = False, 
+                     drawAs='stick', pStyle='cartoon',
+                     propPanel = False, 
+                     colorPanel = False, 
+                     labelPanel = False):
+    
+    """This function initiates required widgets and 3Dmol.js viewer
+    uid : an unique id for 3DMol.js viewer, each of the ipywidgets, and all the global objects
+    moldict : it is a molecule dictionary where is keys are string
+    protein : it is rdkit standard mol object
+    drawAs : representation type for moldict or small molecule
+    pStyle : protein representation style
+    ** the rest of the arguments are panels (user may or may not need those)
+    """
+    
+    if 'ipy_wgs' not in globals():
+        globals()['ipy_wgs'] = dict()
```

Comment:
It seems like you prefer symbol instead of word. I have no objection with it. So, you may expect symbol instead of word in the next commit in every similar cases. Thanks a lot for teaching good code practice.